src/api/XR.js

```diff
@@ -58,11 +58,11 @@ export default class XR extends EventTarget {
     // 'inline' is always guaranteed to be supported.
     if (mode != 'inline') {
       if (!this[PRIVATE].device.supportsSession(mode)) {
-        return Promise.reject(null);
+        return Promise.reject(new DOMException('The specified session configuration is not supported.'));
```

Comment:
This error message is from the latest Chrome Canary. I copied and pasted.

Reply:
I wonder if DOMException constructor needs to be polyfilled anywhere__